CHANGES.md

```diff
@@ -1,3 +1,44 @@
+### 0.10.0 (2017-04-28)
+
+DataKit has a new home: It now lives in the Moby Project!
+
+**all**
+- rename docker/datakit into moby/datakit (#528, @talex5)
+- use jbuilder to build the project (#532, @samoht)
```

Comment:
brief note here on the build speedup and what were the linking bugs you mentioned were fixed @samoht?